scss/core/abstracts/_variables.scss

```diff
@@ -173,6 +173,7 @@ $font-weight-light:           300 !default;
 $font-weight-normal:          400 !default;
 $font-weight-semi-bold:       600 !default;
 $font-weight-bold:            700 !default;
+$font-weight-bolder:          900 !default;
```

Comment:
I think 900 can be black. You can add value 800 for bolder. These are the common font weight name mapping. Let's add all this?

<img width="1440" alt="Screenshot 2020-10-21 at 10 58 19 AM" src="https://user-images.githubusercontent.com/34312966/96677109-85702300-138c-11eb-8dce-b3e40c3f998b.png">

Reply:
but for font lato font-weight:800 is not available and victoria used 900 in page headers.

Reply:
Oh ok.. Then just replace font-weight-bolder with font-weight-black, so that we go according to common practice

Reply:
should i change name into `font-weight-black`?

Reply:
Ya